src/pages/index.js

```diff
@@ -4,13 +4,20 @@ import Link from 'gatsby-link'
 const IndexPage = () => (
   <div>
     <p>
-      We are a group of passionate software developers providing software consulting specialising in full-stack development, distributed systems and serverless applications.
-      Members with over 10 years of experience in the IT provide expertise and services such as architectural reviews and POCs.
-      While relying on the remote work and agile methodologies, we are delivering software projects in the UK, CZ and DE.
+      We are a group of passionate software developers providing software
+      consulting specialising in full-stack development, distributed systems and
+      serverless applications. Members with over 10 years of experience in the
+      IT segment provide expertise and services such as architectural reviews
+      and Proof of Concepts.
+    </p>
+    <p>
+      While relying on the remote work and agile methodologies, we are
+      delivering software projects in the United Kingdom, Czech Republic and
+      Germany.
     </p>
 
-    <h2>Services</h2>
-    <ul>
+    <h3>You can expect us to do</h3>
```

Comment:
Mixing Camel Case and lowercase headers (https://english.stackexchange.com/questions/14/which-words-in-a-title-should-be-capitalized)

Reply:
As long as it is consistent its fine